### DISCUSSION TITLE
Any next release schedule?

### DISCUSSION THREAD

**Discussion by elricym** [ProjectOpenSea/seaport-js#223] [Q&A]
Hi, our product is a bit blocked by the issue of bulk fulfilling criteria orders. I saw it's has been fixed for a long time. So I just curious about the next release date. It will be really helpful if I can use such feature on our product a little bit earlier. 
Thank you Opensea team.

For reference: https://github.com/ProjectOpenSea/seaport-js/pull/188

**Comment by ryanio** [comment]
Sorry about the delay! We just released [v1.2.0](https://github.com/ProjectOpenSea/seaport-js/releases/tag/v1.2.0)
